reports/Certora_Extra_Checks.md

```diff
@@ -0,0 +1,41 @@
+# List Of Checks Done
+
+## proposal 2:
+- on V2:
+    - We checked that the (correct) asset was forzen
+    - We checked that the values that were declared to be updated in the proposal were indeed adjusted to on the correct asset and to the declared values
+    - We checked that no other parameters were changed on this proposal
+    - We checked that the changes were applied on the correct market (v2 and not v3 for example)
+
+- on V3:
+    - We checked that the (correct) asset was forzen
+    - We checked that the values that were declared to be updated in the proposal were indeed adjusted to on the correct asset and to the declared values
+    - We checked that no other parameters were changed on this proposal
+    - We checked that the changes were applied on the correct market (v3 and not v2 for example)
+
+- What we didnt check:
+    - Create2 of real addresses
+    - e2e tests and additional tests made by the proposer
+
+
+## proposal 3:
+- For each market:
+    - checkd that the correct market was called (Ethereum V2 or V3, etc, depending on the payload)
+    - We checked that the values that were declared to be updated in the proposal were indeed adjusted to on the correct asset and to the declared values, i.e. slope 1 was changed to 6% (6_00) 
+    - We checked that no other parameters were changed on this proposal, i.e. all other values were kept the same
+
+
+## Questions:
```

Comment:
The only way is asking the proposer

Reply:
On V3 the engine has the option of KEEP_SAME_VALUE or something along those lines.
On V2 we checked the values on the block explorer to ensure the liquidity bonus is the same as they set there